packages/@tinacms/schema-tools/src/types/index.ts

```diff
@@ -7,6 +7,10 @@ type Meta = {
   error?: any
 }
 
+// This type is used in collectionFilter for reference field, it represent the datatype of the field in the referenced collection that will be used to do the filter
+// This type is extendable if we need to support more type for reference selection filter (e.g. boolean)
+type FilterValue = string[] | string
```

Comment:
This was giving me a type error. Can we change this to something like, 

```
type FilterValue = {[key]: string};
```

Reply:
Done - Fixed the schema, can you pull latest changes and see if the type error still exist?